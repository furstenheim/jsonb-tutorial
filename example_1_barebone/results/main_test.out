CREATE EXTENSION jsonb_tutorial;
SELECT jsonb_tutorial('{}', '{}');
 jsonb_tutorial 
----------------
 
(1 row)

