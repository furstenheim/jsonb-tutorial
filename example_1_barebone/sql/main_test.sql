CREATE EXTENSION pg_conf_eu;
SELECT pg_conf_eu('{}', '{}');