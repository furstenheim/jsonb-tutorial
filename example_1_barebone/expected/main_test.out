CREATE EXTENSION pg_conf_eu;
SELECT pg_conf_eu('{}', '{}');
 pg_conf_eu 
------------
 
(1 row)

